examples/charm++/ckcallback/Makefile

```diff
@@ -18,4 +18,4 @@ reftest.o: reftest.C reftest.decl.h
 	$(CHARMC) -c reftest.C
 
 test: all
-	$(call run, +p4 ./reftest 16 4 )
```

Comment:
Just checking if this is intended. Also, should this example also have a testp target?

Reply:
Yes, that was an intended minor fix because `reftest` doesn't use input arguments, so those values were simply discarded.  

From `examples/charm++/ckcallback/reftest.C`: 
```
  Main(CkArgMsg* msg) {
    delete msg;
    magic_number = 29;
    max_iter = 1000;
    thisProxy.caller();
    thisProxy.recv_in_order();
  }
```